clients/src/main/java/org/apache/kafka/common/metrics/stats/Rate.java

```diff
@@ -115,24 +114,10 @@ private double convert(long timeMs) {
         }
     }
 
-    public static class SampledTotal extends SampledStat {
-
-        public SampledTotal() {
-            super(0.0d);
-        }
-
-        @Override
-        protected void update(Sample sample, MetricConfig config, double value, long timeMs) {
-            sample.value += value;
-        }
-
-        @Override
-        public double combine(List<Sample> samples, MetricConfig config, long now) {
-            double total = 0.0;
-            for (Sample sample : samples)
-                total += sample.value;
-            return total;
-        }
-
+    /**
+     * @deprecated since 2.4 Use {@link Sum} instead.
+     */
+    @Deprecated
+    public static class SampledTotal extends Sum {
```

Comment:
This metric was redundant with Sum (their implementations are exactly the same). I decided in favor of Sum because it's a top-level class.

Since this class is public and not internal, though, I figure we'd better just mark it as deprecated for now, although it's unused in our code base as of this PR.